### DISCUSSION TITLE
project name

### DISCUSSION THREAD

**Discussion by tnielens** [pityka/saddle#383] [General]
I noticed https://github.com/saddle/saddle/issues/88. @pityka would you be interested in contacting the original author and ask whether you could become an administrator of the saddle github organization?

**Comment by pityka** [comment]
In response to that request, the original author archived saddle/saddle so I interpreted this as they don't want to bother with a transition. I am all in for the bazaar model of open source software, so happy with the current setup. That said, we can make a second try for the saddle org and if that fails create a new org (with new saddle2 or whatever) so that e.g. your contributions are not tied to my personal account

**Reply by tnielens** [reply]
I'm totally fine with contributing to a repo under your user. But I'd like to promote a bit your fork in my company and maybe outside of it. I fear the saddle organization being archived might confuse or scare potential users as it's the first github repo people open before finding the fork, if they find it. Best option would be you becoming admin of the saddle org. If you agree so, I can send a mail to the original author asking again. If there is no option to get access the organization, I would suggest renaming the project for improved discoverability.

**Reply by pityka** [reply]
This plan is ok for me. do you mind try contacting them again? Thank you!
